### PR TITLE
added some changes

### PR DESCRIPTION
**Pull request by sarahmehri**

### CODE REVIEW

**Comment by sarahmehri** [comment]
added the sound button for media element, filtering the songs by genre and changing the grid view of platform.

**Comment by zewditu** [comment]
There is no way back to all music lists one er start to filter 
image

**Comment by sarahmehri** [comment]
yes, you are right, I will work on it by tomorrow.

**Comment by zewditu** [comment]
And the speaker symbol should be displayed once we started to play the music , otherwise it should be hidden

**Comment by zewditu** [comment]
And the speaker thing should be displayed in each music

**Comment by sarahmehri** [comment]
Tomorrow will work on :)

File: music-library-team2/Model/GenreMenu.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace music_library_team2.Model
+{
+   public class GenreMenu
+    {
+
+        
+        public Genre GenreCategory { get; set; }
```
Comment (zewditu) [inline music-library-team2/Model/GenreMenu.cs:13]: why do you need to define this class?

File: music-library-team2/MainPage.xaml.cs [hunk 2]
```diff
@@ -136,5 +147,13 @@ private void GridView_ItemClick(object sender, ItemClickEventArgs e)
             var music = (Music)e.ClickedItem;
             SoundMedia.Source = new Uri(music.FilePath);
         }
+
+        private void GenrList_ItemClick(object sender, ItemClickEventArgs e)
+        {
+            var menuGenre = (GenreMenu)e.ClickedItem;
+            MusicManager.GetMusicsByGenre(Musics, menuGenre.GenreCategory);
```
Comment (zewditu) [inline music-library-team2/MainPage.xaml.cs:154]: why you did not use genre instead of menue

File: music-library-team2/Model/GenreMenu.cs (hunk 1 above)
Comment (sarahmehri) [reply music-library-team2/Model/GenreMenu.cs:13]: To filter the music by their genre and display every song  by their genre, when click on it

**Pull request merged**